### PR TITLE
Move CONTRIBUTING.md to .github, update Python 2.7 to latest, & change Github releases link to Appveyor artifacts

### PR DESCRIPTION
**Pull request by Margen67**
All URLs should probably be updated to be https not http.

### CODE REVIEW

**Comment by resetnow** [comment]
Will github pick up CONTRIBUTING.md automatically? There is this notification which is shown when an issue is created:

>Please review the <u>guidelines for contributing</u> to this repository.

**Comment by Margen67** [comment]
@resetnow I went into the new issue creator on Github and the links work fine on my repo.
Github should automatically pick it up since it's one of the intended locations as shown here: https://help.github.com/articles/setting-guidelines-for-repository-contributors/#adding-a-contributing-file

I moved it there for better organization.

File: README.md [hunk 1]
```diff
@@ -90,5 +90,5 @@ For more see the main [frequently asked questions](http://xenia.jp/faq/) page.
 
 ### Can I get an exe?
 
-Check the [Releases](https://github.com/benvanik/xenia/releases) tab to see
-what's there. Updated irregularly.
+Check the [Artifacts](https://ci.appveyor.com/project/benvanik/xenia/build/artifacts) tab to see
```
Comment (DrChat) [inline README.md]: Not a tab anymore - say "Check Appveyor's artifacts to see what's there."

**Comment by DrChat** [comment]
Thanks for the changes - been meaning to do some of these myself. 👍

File: README.md (hunk 1 above)
Comment (Margen67) [inline README.md]: Done.

**Pull request merged**